src/script.css

```diff
@@ -1,17 +1,91 @@
+.supp-corner {
+  --main-cl: #bfc;
+  --bg: #fff;
+  --me-cl: rgb(226, 255, 233);
+  --them-cl: #efb;
+  --gap: 7px;
+  --shadow-cl: #888; /* should look ok on dark and light backgrounds ;) */
+}
```

Comment:
this section can be used to fiddle with the looks without risking breaking the layout css